HitDelay.py

```
@@ -1,429 +1,429 @@
+			self.delayHistory.insert('', END, values=(ids[0],ids[1],ids[3],'%.6f ms'%ids[2],'%.6f ms'%ids[4]))
+		del data
+		self.UpdateWindowInfo()
+
+	def Draw(self):
+		consoleFind = False
+		try:
+			win = uiauto.WindowControl(searchDepth=1,Name='MUSYNX Delay',searchInterval=1).DocumentControl(serchDepth=1,Name='Text Area',searchInterval=1)
+			win.SendKeys('{Ctrl}A',waitTime=0.1)
+			win.SendKeys('{Ctrl}C',waitTime=0.1)
+			consoleFind = True
+		except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)